wolfcrypt/src/random.c

```diff
@@ -2103,17 +2104,17 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
       defined(WOLFSSL_CONTIKI)
 
     /* these platforms do not have a default random seed and
-       you'll need to implement your own wc_GenerateSeed or define via
+       you'll need to implement your own GenerateSeedInternal or define via
        CUSTOM_RAND_GENERATE_BLOCK */
 
     #define USE_TEST_GENSEED
 
 #elif defined(NO_DEV_RANDOM)
 
-    #error "you need to write an os specific wc_GenerateSeed() here"
+    #error "you need to write an os specific GenerateSeedInternal() here"
 
     /*
-    int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+    static int GenerateSeedInternal(OS_Seed* os, byte* output, word32 sz)
```

Comment:
Sean, won't this break compatibility with folks who have their own `wc_GenerateSeed` defined? Same with the `WOLFSSL_CONTIKI case above. I like the refactor, but can we do it such that it only occurs when `WOLF_CRYPTO_CB` is defined to keep better backwards compat?

Reply:
Returned APIs and put PKCS11 calls into a couple of places.